qiita_db/analysis.py

```diff
@@ -266,6 +275,21 @@ def name(self):
         return conn_handler.execute_fetchone(sql, (self._id, ))[0]
 
     @property
+    def portal(self):
```

Comment:
Please do not use double underscores as they act unexpectedly. A single underscore is fine. If this is a debug only method, it could include a check that raises if the environment is not in debug mode.

```python
> class foo(object):
   ....:     def __bar(self):
   ....:         print self
   ....:

16:04:29 (mcdonadt@Daniels-MacBook-Pro-3):/Users/mcdonadt/ResearchWork/greengenes_release/gg_13_8_otus/trees
> x = foo()

16:04:31 (mcdonadt@Daniels-MacBook-Pro-3):/Users/mcdonadt/ResearchWork/greengenes_release/gg_13_8_otus/trees
> x.__bar()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-14-883553e2ea7d> in <module>()
----> 1 x.__bar()

AttributeError: 'foo' object has no attribute '__bar'
```